<launch>
  <!--
    Push the left frontal button labeled as 'RB' to activate cmd_vel publishing.
    Move the right stick around to control the velocity.
   -->

  <!-- joy node -->
  <node respawn="true" pkg="joy" type="joy_node" name="joystick" >
    <param name="dev" type="string" value="/dev/input/js4" />
    <param name="deadzone" value="0.12" />
  </node>
  
  <!-- Axes -->
  <node pkg="teleop_keyandjoy" type="teleop_joy" name="teleop_joystick" output="screen">
    
    <param name="scale_angular" value="0.4"/>
    <param name="scale_linear" value="0.4"/>
    <param name="max_angular" value="0.8"/>
    <param name="max_linear" value="0.8"/>
    <param name="axis_actbtn" value="4"/>
    <param name="axis_linear" value="3"/>
    <param name="axis_angular" value="2"/>
  
  </node>

</launch>
